src/parsers/perf/perfparser.cpp

```diff
@@ -503,6 +503,33 @@ struct SymbolData
     }
 };
 
+struct CallerCalleeLocation
+{
+    QString symbol;
+    QString binary;
+
+    bool operator<(const CallerCalleeLocation& location) const
+    {
+        return std::tie(symbol, binary)
```

Comment:
join with next line (or is it too long? in that case, one more space in the line below, so that the `std::tie` aligns